includes/fbbot/class-omise-fbbot-user-service.php

```diff
@@ -0,0 +1,86 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+if ( class_exists( 'Omise_FBBot_User_Service' ) ) {
+  return;
+}
+
+class Omise_FBBot_User_Service {		
+	private function __construct() {
+		// Hide the constructor
+	}
+
+	public static function get_user( $sender_id ) {
+		$user = get_user_by( 'login', $sender_id );
+
+		if ($user) {
+			return $user->data;
+		}
+
+		$fb_profile = self::get_fb_profile( $sender_id );
+		if ( ! $fb_profile ) 
+			return NULL;
+
+		$wp_user = self::create_wp_user( $fb_profile );
```

Comment:
Got it 🙇 ,  I create a new user because don't want to get a data from facebook user every time when the old user use the bot. Should we let them to be guest or not ? :bowtie: